Controllers/tutorials.js

```diff
@@ -0,0 +1,39 @@
+const getAllTutorials = (req, res) => {
+  res.json({ message: `Path: ${req.path}, Method: ${req.method}` });
+};
+
+const getSingleTutorial = (req, res) => {
+  res.json({
+    message: `Path: ${req.path}, Method: ${req.method}, Id: ${req.params.id}`,
+  });
```

Comment:
You might wanna check, if there is a `req.params.id` or is this a number or a string?